<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKDebuggerPluginTimedSendingItem</class>
 <widget class="QWidget" name="SAKDebuggerPluginTimedSendingItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Timed sending</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="9">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Suffix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Interval</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="enableCheckBox">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Writing data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="descriptionLineEdit">
     <property name="text">
      <string notr="true">Item</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QComboBox" name="suffixComboBox"/>
   </item>
   <item row="0" column="7">
    <widget class="QComboBox" name="textFormatComboBox"/>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Text format</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="intervalSpinBox">
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="10">
    <widget class="QLineEdit" name="dataLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
